Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OuterTune/App](https://hosted.weblate.org/projects/outertune/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/outertune/app/horizontal-auto.svg)

lucasmz-dev commented:
Weblate may push when new translations are added (translation files), there's actually more changes happening in the meanwhile, two people are helping :)

lucasmz-dev commented:
@mikooomich fyi I've asked Izzy on their issue tracker to have the translation link added in their repo

lucasmz-dev commented:
@mikooomich I updated the repo before the force push, might have to rebase manually

lucasmz-dev commented:
Nevermind, it went alright!

Or not... The commits removed are still here.

lucasmz-dev commented:
doing a rebase

lucasmz-dev commented:
weblate needs to stay locked until #257 is merged and the weblate repo is reset

mikooomich commented:
merged

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/

lucasmz-dev commented:
alright everything seems ok

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/

Kefir2105 committed:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/

Commit:
Translated using Weblate (Italian)

Currently translated at 84.9% (107 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/it/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/it/

Kefir2105 committed:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (126 of 126 strings)

Translation: OuterTune/App
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/

mikooomich commented:
@lucasmz-dev when merging this, are there anythings to look out for? For example, squash merging vs standard merge

lucasmz-dev commented:
@mikooomich I don't think it matters much, the difference is how people will end up being credited in the commits as it'd be less specific. 

Weblate has a add on you can add that squash commits depending on certain rules, e.g. you can squash commits for the same languages, or author, or just do it all in a single commit. I didn't enable this here because it can be a little bit dangerous since you'll always lose a bit of the history in some way or another and that's for you to decide and it can behave weirdly if you edit it more than once.

One care I'd do is if you modify any strings (adding new ones only should be fine) I'd commit any awaiting changes and merge Weblate beforehand, and maybe lock Weblate for a while in case you find anything weird.
